src/course-checklist/data/api.js

```diff
@@ -0,0 +1,75 @@
+// @ts-check
+import { camelCaseObject, getConfig } from '@edx/frontend-platform';
```

Comment:
typically I believe that things like api calls are the most important pieces of code to test in an application.